### PR TITLE
[WIP] modified spec xsl for new copyright

### PR DESCRIPTION
**Pull request by himorin**
for copyright starting from 1st Jan. 2023
I will check with pubrules, whether this modification is ok per guide or not.

### CODE REVIEW

**Comment by himorin** [comment]
will check using pubrules after 1st Jan. 2023.

**Comment by himorin** [comment]
updated process document from 2020 to 2021, and updated copyright line (see also [specberus issue](https://github.com/w3c/specberus/pull/1687#event-8134581423))

File: spec/xmlspec.xsl [hunk 1]
```diff
@@ -1111,28 +1111,13 @@
...
-            <a href="http://ev.buaa.edu.cn/">Beihang</a>
-            <xsl:text>). W3C </xsl:text>
+            <a href="https://www.w3.org/">World Wide Web Consortium, Inc.</a>. 
+            <abbr title="World Wide Web Consortium">W3C</abbr><sup>&#xae;</sup>
             <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>
             <xsl:text>, </xsl:text>
             <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a>
             <xsl:text> and </xsl:text>
-            <a href="https://www.w3.org/Consortium/Legal/copyright-documents">document use</a>
+            <a href="https://www.w3.org/Consortium/Legal/copyright-software">permissive document license</a>
```
Comment (nigelmegitt) [inline spec/xmlspec.xsl:1120]: I don't recall us agreeing to adopt the permissive document license - is there a pointer to the decision behind this, please?

**Review by nigelmegitt — APPROVED** [review]
LGTM, thank you @himorin .

**Comment by himorin** [comment]
passed pubrules, so merging this.

**Pull request merged**